lib/utils/index.ts

```diff
@@ -1,5 +1,4 @@
-import {
-    aws_lambda as lambda,
-  } from "aws-cdk-lib";
+import { aws_lambda as lambda } from "aws-cdk-lib";
 
 export type CustomLambdaFunctionProps = lambda.FunctionProps | any;
+export const DEFAULT_PGSTAC_VERSION = "0.9.5";
```

Comment:
Would a shared requirements file seem too heavy for this usecase?

Reply:
I think that would be too much since we want users to be able to configure the pgstac version for the database (and now the ingestor)

Reply:
Managing versions in a utils file seems a little strange to me, but I could see how having it added to constraints keeps the requirements file clean

Reply:
yeah, I agree in general but this is a simple way to define a default value for the `pgstacVersion` parameters from multiple constructs in a single place!